### ISSUE TITLE
Move default_runtime_attributes specified in the options.json file to the wdl task level

### ISSUE THREAD

**Issue by aclum** [microbiomedata/mg_annotation#11]
We've decided to move away from specifying runtime attributes in an options.json, instead defaults should be specified in the wdl at the task level. JAWS does not and will not support specifying a cromwell options.json because they want flexibility to change the workflow runner in the future.

**Comment by aclum** [comment]
This is current resolved in the werkflow branch.
